src/get-started/install/_get-sdk.md

```diff
@@ -51,7 +51,7 @@ Run the following command to see if there are any dependencies you need to
 install to complete the setup:
 
 ```terminal
-$ flutter doctor
+$ flutter doctor [-v]
```

Comment:
See earlier comment about this.

Reply:
This point wasn't addressed.

Reply:
Well, this PR seemed to be messed up, too. So annoying.